docs/configuring-playbook-appservice-draupnir-for-all.md

```diff
@@ -0,0 +1,87 @@
+# Setting up Draupnir for All/D4A (optional)
+
+The playbook can install and configure the [Draupnir](https://github.com/the-draupnir-project/Draupnir) moderation tool for you in appservice mode.
+
+Appservice mode can be used together with normal Draupnir or independently. 
```

Comment:
```suggestion
Appservice mode can be used together with the regular [Draupnir bot](configuring-playbook-bot-draupnir.md) or independently. 
```

Reply:
Also, adding some information on pros/cons to using bot vs appservice, would be nice.

The appservice is more multitenant? It's also not rate-limitted? Etc.

Reply:
Appservice is Multitennant yes and Ratelimit Exempt just like a Normal Draupnir should be. Will add this to the documentation.